tcl/return.tcl

```diff
@@ -50,10 +50,10 @@ proc qc::return2client { args } {
     # Content-Disposition
     if { $download } {
         # Download payload as an attachment
-        default content-disposition "attachment; filename=$filename"
+        default content-disposition "attachment; \"filename=$filename\""
```

Comment:
What does this structured string say we should do if $filename contains a quote ?